packages/api-cardano-db-hasura/schema.graphql

```diff
@@ -611,7 +611,7 @@ type Reward {
   address: StakeAddress!
   amount: String!
   earnedIn: Epoch!
-  receivedIn: Epoch!
+  receivedIn: Epoch
```

Comment:
Could you add test for the case when receivedIn is null/undefined?

Reply:
Not at the moment, unfortunately, as we only have API tests against _testnet_ where we cannot control this state. It' 's a transient state, so apps just need to guard against it being undefined, which is all this change does.